content/docs/guides/components/istio.md

```diff
@@ -60,16 +60,18 @@ so this service should not be of type `LoadBalancer`.
 - Edit `install/kubernetes/istio-demo.yaml` and change the service type of `istio-ingressgateway` from
   `LoadBalancer` to `NodePort`.
 
-### 4. Install istio (without mTLS)
+### 5. Install istio (without mTLS)
     ```
     kubectl apply -f install/kubernetes/istio-demo.yaml
     ```
 
-### (optional) 5. Deploy the Gateway
```

Comment:
Why is deploying the gateway a separate step? It's all one dependency, right?

Reply:
gateway it's a client resource, not part of the installation.
The installation comes with a dummy gateway in istio's namespace.
Here we are creating one that will be used by kubeflow in kubeflow namespace